components/layout/navigation.tsx

```diff
@@ -51,6 +58,7 @@ export function Navigation() {
                         : "text-gray-600 hover:text-gray-700",
                       "hover:bg-blue-100",
                     )}
+                    target={item.href.includes("https") ? "_blank" : undefined}
                   >
```

Comment:
We shouldn't modify the behaviour of `next/link` like this. We should render a standard link with a `target="_blank"` if the `external` config above is set to true.

Reply:
ahh I see okay got it changing it in a sec.